front-end/src/App.test.js

```diff
@@ -24,10 +27,12 @@ describe('App', () => {
     });
     
     it('should display AnalysisPage when View Analysis button is pressed on MainPage', async() => {
-        store.dispatch(setTranscriptUploadStatus(true))
+        store.dispatch(setTranscriptUploadStatus(true));
+        store.dispatch(setProjectIdToBeAnalyzed("1"));
+        store.dispatch(addAnalyzedTranscript({projectId: "1", transcript: [{question: "a", children: {"b": 100,}}]}));
```

Comment:
Maybe label what this does in a concrete sense (i.e. 
"represents when a transcript has been uploaded")